webapp/src/plugin_constants/messages.ts

```diff
@@ -7,4 +7,10 @@ export const error = {
 
     // Subscription
     subscriptionAlreadyExists: 'Requested subscription already exists',
+    accessDenied: 'Access Denied',
+    subscriptionNotFound: 'Requested subscription does not exists',
+    adminAccessError: 'Looks like you do not have access to add/delete subscription for this project. Please make sure you are a project or team administrator for this project',
+
+    // Link
+    notAccessibleError: 'Looks like this project/organization does not exists or you do not have permissions to access it',
```

Comment:
```suggestion
    notAccessibleError: 'Looks like this project/organization does not exist or you do not have permissions to access it.',
```

Reply:
we haven't put any full stop after any message on webapp
also, its a standard practise across many products that they avoid `.` for message